### ISSUE TITLE
Kotlin support

### ISSUE THREAD

**Issue by Aaur1s** [Liubsyy/JarEditor#1]
Plugin is cool, so Kotlin support would be really cool

**Comment by Liubsyy** [comment]
It is easy to compile a new kotlin file into a class, but it is difficult to decompile a class into kotlin. Class is usually decompiled into java.

**Comment by Aaur1s** [comment]
Yeah I know that, that's not what I meant by Kotlin support. Intelij opens Kotlin stubs when you're trying to use this plugin on Kotlin classes and you can't click decompile to java for typing code. Sorry for confusion

**Comment by Liubsyy** [comment]
Got it, the next version can support it. At present, you can copy the decompiled java code into the Kotlin class decompiled editor.

**Comment by Liubsyy** [comment]
v1.4 supports kotlin
